core/arch/aarch64/instr_create.h

```diff
@@ -277,6 +277,16 @@
     opnd_add_flags((sht), DR_OPND_IS_SHIFT), (sha))
 #define INSTR_CREATE_b(dc, pc) \
   instr_create_0dst_1src((dc), OP_b, (pc))
+
+/**
+ * This INSTR_CREATE_xxx macro creates an instr_t with opcode OP_xxx and
+ * the given explicit operands, automatically supplying any implicit operands.
```

Comment:
The two line above seems general and not specific to this new INSTR_CREATE_bl. Kind of strange.
Could you please make it more specific to this instr_create, e.g. creates an branch & link ...
Xref other INSTR_CREATE... with examples.
Or no comment, as the rest ones without comment.